src/main/java/rick/Rick.java

```diff
@@ -0,0 +1,52 @@
+package rick;
+
+import rick.command.Command;
+
+/**
+ * The main class to interface with the Rick server.
+ *
+ * @author SeeuSim
+ *         AY2223-S2 CS2103T
+ */
+public class Rick {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private boolean isTimeToExit = false;
+
+    /**
+     * Default constructor to set up the app.
+     */
+    public Rick() {
+        this.storage = Storage.create();
+        this.ui = new Ui();
+        this.tasks = TaskList.create(this.storage, this.ui);
+    }
+
+    /**
+     * Generate the server UI welcome message.
+     * @return The message.
+     */
+    public String getWelcome() {
+        return ui.welcomeMessage();
+    }
+
+    /**
+     * Process the command and return the server UI response.
```

Comment:
Perhaps add a new line after the description, before params to increase readability